doc/nrf/libraries/bluetooth_services/services/latency.rst

```diff
@@ -24,7 +24,7 @@ Latency Characteristic (``67136E02-58DB-F39B-3446-FDDE58C0813A``)
 =================================================================
 
 Write
-   The server is waiting for a write request from the client, and replies with a write response.
+   The server is waiting for a ``write`` request from the client, and replies with a ``write`` response.
```

Comment:
Consider the following:
```suggestion
   The server is waiting for a ``write request`` from the client, and replies with a ``write response``.
```

Reply:
I wonder if we shouldn't treat whole `write request`/`write response` as a single term here. I think it may be more intuitive.

Reply:
How about just without emphasis?

Reply:
Would be ok for me too.